nrf_wifi/hw_if/hal/src/hal_api.c

```diff
@@ -1353,6 +1386,34 @@ void nrf_wifi_hal_dev_deinit(struct nrf_wifi_hal_dev_ctx *hal_dev_ctx)
 }
 
 
+enum nrf_wifi_status nrf_wifi_hal_lock_rx(struct nrf_wifi_hal_dev_ctx *hal_dev_ctx)
+{
+	enum nrf_wifi_status status = NRF_WIFI_STATUS_FAIL;
```

Comment:
Why do we need the status variable here? Maybe just make the function have a `void` return?